### DISCUSSION TITLE
Is it okay to PowerPlatform-DataverseServiceClient with Azure Function V4 targeting .NET 6.0?

### DISCUSSION THREAD

**Discussion by twiga2013** [microsoft/PowerPlatform-DataverseServiceClient#202] [General]
Is it okay to use PowerPlatform-DataverseServiceClient with Azure Function V4 targeting .NET 6.0?

**Comment by twiga2013** [comment]
Can someone from Microsoft please let me know if the Dataverse Client SDK can used targeting .NET 6.0?

**Reply by MattB-msft** [reply]
It works properly with .net 6.  we will be updating our readme shortly to that effect now that .net 6 has shipped.

**Comment by RichieRoxx** [comment]
Although you might be looking for a official approval that it's production ready, which I can cannot give you, I can at least confirm that it works with .NET 6 in an AzureFunction 4.x. I can verify that connection, read, query and update worked for me. The fact, that it has a version number leading with 0 might also be an indicator for you how "production ready and tested" it is. I hope work on this module speeds up a lot soon. I think it's about time.
